lib/tarjan_tree.rb

```diff
@@ -1,24 +1,27 @@
 # Class for using the Tarjan algorithm to remove cycles in the symbol trees.
 class TarjanTree
   include TSort
-  attr_accessor :hashed_symbols, :subclasses, :ontology
+  attr_accessor :hashed_symbols, :subclasses, :ontology, :symbols, :subclass_sym
```

Comment:
Please use a plural named variable like `subclass_symbols` instead of `subclass_sym`.